### ISSUE TITLE
Constantly crashes. Reloads, crashes, repeat.

### ISSUE THREAD

**Issue by Xeddicus** [gorhill/uBlock#3072]
### Describe the issue
Fresh install of the extension and it crashes immediately. Click to reload, crashes again. Repeat.

### One or more specific URLs where the issue occurs
Soon as browser opens, so no URL.

### Steps for anyone to reproduce the issue
Install extension, it crashes.

### Your settings

- OS/version: Windows 10 1607
- Browser/version: Chrome Version 63.0.3223.8 (Official Build) dev (64-bit)
- uBlock Origin version: 1.14.8 (and a few previous versions tried as well)

##### Your filter lists
Default

##### Your custom filters (if any)
None.

**Comment by jspenguin2017** [comment]
Canary builds can blow up at any time, everything works as intended on the latest stable release for me.

**Comment by Mstrodl** [comment]
Can reproduce on `macOS 10.12.3 (16D32)`
Chrome Canary `63.0.3227.0 (Official Build) canary (64-bit)`
uBlock `1.14.8`

**Comment by gorhill** [comment]
Any browser crash must be reported to Chromium devs. uBO is HTML/CSS/JS, so it's not supposed to crash<sup>[1]</sup> -- it's a browser issue.

<sub>[1] @jspenguin2017 Yes, I know, let's not split hair here please.</sub>

**Comment by gorhill** [comment]
For any crash experienced with a dev build of a browser, please always see if you can reproduce using the stable release of the browser.

By the way, I do have Chrome 63.0.3223.8 (Official Build) dev (64-bit) (Linux), and I am not experiencing any crash.

**Comment by ivnax** [comment]
Running version 62.0.3202.31 (Official Build) dev (64-bit) on a Chromebook and I am getting the same crash

uBlock 1.14.8

**Comment by uBlock-user** [comment]
Can't reproduce on **63.0.3228.0 (Official Build) (64-bit)**

**Comment by cole-h** [comment]
Experienced the same crash loop on: Version 63.0.3223.8 (Official Build) dev (64-bit)

I fixed it by launching Chrome with the "--disable-appcontainer" switch. The crash was present on 1.14.8, 1.14.10, and 1.14.11.110 (dev build). From a quick preliminary search around the web, I could not find any way to discover why it was crashing. I also tried packing the extension on my own, installing the zip, etc. for all versions tested.
